tests/unit/pretender-query-param-handler-test.js

```diff
@@ -436,9 +436,47 @@ module('pretender-query-params-handler', function () {
 
       await assert.rejects(
         fetch('/api/graphql?foo=abc&bar=xyz&var=999'),
-        /Pretender intercepted GET \/api\/graphql\?foo=abc&bar=xyz&var=999 but no handler was defined for this type of request/,
+        /Pretender intercepted GET \/api\/graphql\?foo=abc&bar=xyz&var=999 \nbut found no handler for it/,
         'should return an error message'
       );
     });
   });
+
+  module('better error message', function (hooks) {
```

Comment:
can we add a test where

0. the query param names do match and
1. the values don't match and
2. there is a fallback handler

and have this case throw the useful error (rather than fall back to the fallback handler)

Reply:
Added a test case as suggested.